cmd/resim/commands/project.go

```diff
@@ -205,7 +189,11 @@ pageLoop:
 			if project.ProjectID == nil {
 				log.Fatal("project ID is empty")
 			}
-			if *project.Name == projectName {
+			if *project.Name == identifier {
+				projectID = *project.ProjectID
+				break pageLoop
+			}
+			if project.ProjectID.String() == identifier {
```

Comment:
I think this could be more efficient if you check if the identifier parses as a uuid then use a "get" - then you don't have to page through potentially very many. 

In the future the API will have a GetProjectWithName query anyway, so that's future compatible

Reply:
Yes good idea. Switched to this...